distrib-dl

```diff
@@ -223,6 +227,22 @@ function download_gentoo_live() {
     cd "$download_dir/gentoo/" && sha512sum --ignore-missing -c <(cat ${gentoo_sums_filename}) && cd -
 }
 
+download_opensuse_tumbleweed() {
+    # https://www.opensuse.org/#Tumbleweed
+    opensuse_base_url="http://opensuse.mirrors.theom.nz/${opensuse_release}/iso/"
+    opensuse_iso_filename="${opensuse_image}.iso"
+    opensuse_sums_filename="${opensuse_iso_filename}.sha256"
+    if [[ ! -d "$download_dir/opensuse/" ]]; then mkdir -p "$download_dir/opensuse/"; fi
+    echo "[distrib-dl] INFO: Downloading Opensuse Tumbleweed keyring, checksums and signature..."
+    gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys "$opensuse_gpg_fingerprint"
```

Comment:
Hi @unixbhaskar sorry for the delay, I am checking your patches now.

I would prefer to `wget` the opensuse key at https://download.opensuse.org/tumbleweed/repo/oss/gpg-pubkey-3dbdc284-53674dd4.asc (as described on https://software.opensuse.org/distributions/tumbleweed) instead of getting it from a third party hkp server. Then import it with `gpg --import` (consistency with other distributions in the script, follows official documentation)